.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| Critical? | no
```

Comment:
Not sure if this should be really here. I don't want to encourage false critical issues. Maybe this should be removed and Sulu core members can set the label, and not core members can add a critical line themself if they really think it is critical and are more involved with contributions.